core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -244,7 +244,7 @@ private TableProperties() {}
   public static final String OBJECT_STORE_ENABLED = "write.object-storage.enabled";
   public static final boolean OBJECT_STORE_ENABLED_DEFAULT = false;
 
-  // Excludes the partition values in the path when set to true and object store is enabled
+  // Excludes the partition values in the path when set to false and object store is enabled
```

Comment:
Nit: While this comment is correct, I feel like writing the comment when the property is true reads a little bit more easy.

```
// Includes the partition values in the path when set to true and object store is enabled
```

Reply:
+1 to changing the comment as @amogh-jahagirdar suggested